examples/custom-generator-codelab/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "sample-app",
+  "version": "1.0.0",
+  "description": "A sample app using Blockly",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve --open"
+  },
+  "keywords": [
+    "blockly"
+  ],
+  "author": "Blockly team",
```

Comment:
General file comment: if this is specific to the custom generator codelab, change the name and description. If you are deliberately leaving it as the general form, remove "author".